SpatialGDK/Source/Private/Interop/SpatialReceiver.cpp

```diff
@@ -909,8 +885,19 @@ void USpatialReceiver::QueueIncomingRPC(const TSet<FUnrealObjectRef>& Unresolved
 void USpatialReceiver::ResolvePendingOperations_Internal(UObject* Object, const FUnrealObjectRef& ObjectRef)
 {
 	UE_LOG(LogSpatialReceiver, Log, TEXT("Resolving pending object refs and RPCs which depend on object: %s %s."), *Object->GetName(), *ObjectRef.ToString());
-	Sender->ResolveOutgoingOperations(Object, /* bIsHandover */ false);
-	Sender->ResolveOutgoingOperations(Object, /* bIsHandover */ true);
+
+	FClassInfo* Info = nullptr;
+	if (AActor* Actor = Cast<AActor>(Object->GetOuter()))
+	{
+		Info = TypebindingManager->FindClassInfoByClassAndOffset(Actor->GetClass(), ObjectRef.Offset);
```

Comment:
This part is wrong. `Object` is the value that has been resolved, not the object that was replicating that property, so the class info of this object doesn't matter.
The info should be found in the `ResolveOutgoingOperations` for the `ReplicatingObject`.